### DISCUSSION TITLE
Error migrating to codemagic.yaml file

### DISCUSSION THREAD

**Discussion by tomassasovsky** [codemagic-ci-cd/codemagic-docs#2154] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Open an app workflow that's configured with the Workflow Editor. The repository is a private bitbucket repo.
2. Tap the "Switch to codemagic.yaml" button

### Expected results

The codemagic.yaml file should be populated with the current configuration for the app.

### Actual results

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/63405252/df1c38d7-6416-44c1-85a5-c3c9b92d83cb)
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/63405252/ca7d0c23-59ba-4159-b2cc-549dde4aab01)

**Comment by himesh-cm** [comment]
Hi @tomassasovsky,

It is not possible to convert wfe to yaml config.

​You can use the sample project yaml file, which should work with minimal changes. You can refer to it [here](https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main/flutter/flutter-android-and-ios-yaml-demo-project).

After switching to the `codemagic.yaml` file, a screen will appear, as shown below. Once you add the `yaml` file to the root of your repository, it will automatically populate here.

<img width="1422" alt="Screenshot 2023-10-27 at 11 59 04" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/106069365/1c602884-0e24-4cf1-8d42-3b615ca034cb">

​
Feel free to let me know if you have any queries.

**Comment by himesh-cm** [comment]
Hi @tomassasovsky, I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
